Makefile

```diff
@@ -45,4 +45,10 @@ github.traction:
 	npm run traction
 
 dev:
-	vercel dev --listen 8080 --local-config vercel-local.json
\ No newline at end of file
+	vercel dev --listen 8080 --local-config vercel-local.json
+
+# Clones all migrations from our github repo and copies them into the docker folder
+docker.migrations:
```

Comment:
new script to copy migrations. you can run `make docker.migrations` at any time to grab the latest